build-system/tasks/runtime-test/index.js

```diff
@@ -400,9 +404,24 @@ async function runTests() {
             ? 'beta' : 'stable']
           .push(browserId);
     }
+
+    let totalStableSuccess = 0;
+    let totalStableFailed = 0;
+    const partialTestRunComplete = async(browsers, results) => {
+      if (results.error) {
+        // TODO(danielrozenberg): the last batch fails with an error because of
+        // IE 11 failed tests. Uncomment this once there issues are resolved.
+        // await reportTestErrored();
+      } else {
+        totalStableSuccess += results.success;
+        totalStableFailed += results.failed;
+      }
```

Comment:
Will the above block work if there's another browser with passing tests in the same batch as IE 11? Or will it ignore the count for that browser? For now, would it make sense to remove the `if` block?

Reply:
Good catch. I'll remove the if clause and replace it with a long comment that explains what the behaviour is and what it should be ideally